src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -136,32 +136,37 @@
 
 //Excel returns "$"
 >> Text(Blank(), "$ #,###")
-""
+Blank()
 
 >> Text(Blank(), "#,#")
-""
+Blank()
 
 >> Text(Blank(), "#,###")
-""
+Blank()
 
 >> Text(123.4, Blank())
-""
+Blank()
 
 //Excel throws error
 >> Text(DateTimeValue(Blank()), DateTimeFormat.LongDate)
-""
+Blank()
 
 //Excel throws error
 >> Text(DateTimeValue("March 30, 2022 5:30 PM"), Blank())
-""
+Blank()
 
 >> Text(Date(2016,1,31), "pqr")
 "pqr"
 
-
 >> Text(1/0, "#.000")
 Error({Kind:ErrorKind.Div0})
 
+>> Text(If(1<0,2))
+Blank()
+
+>> Text(ParseJSON("null"))
```

Comment:
We already have this test in ParseJson.txt, no need to repeat it

Reply:
remove this.